test/provision/k8s_install.sh

```diff
@@ -126,7 +122,7 @@ esac
 
 #Install kubernetes
 case $K8S_VERSION in
-    "1.7"|"1.8"|"1.9"|"1.10"|"1.11")
```

Comment:
TBH, I would keep this one. Maybe we drop support, but if it's quite useful in case that we need to provide support to any old cilium version.

Reply:
@eloycoto this PR won't be backported, if we need to test against an old cilium version the test/ directory of that particular cilium version still contains the 1.7 setup